### DISCUSSION TITLE
How to import the JSON annotations file

### DISCUSSION THREAD

**Discussion by russell-mccullin** [tecoholic/ner-annotator#109] [Q&A]
So I use the tool to annotate my text and then export it to JSON.  Is there anyway to import the exported file back into the tool?

If not, then what is the expected format for import Classes and Import Annotations? 

Thank you so much for this great tool

**Comment by alvi-khan** [comment]
Hello! It's possible to import the .json file by first opening the base text file and then importing the .json file using the menu option.
![289535308-07dca972-fc20-4373-98e6-4de8ec65d62c](https://github.com/tecoholic/ner-annotator/assets/55395950/268abed8-a18e-4fcb-8e8d-0c7c2ef45efb)
